code/datums/mind.dm

```diff
@@ -88,6 +88,20 @@
 	return language_holder
 
 /datum/mind/proc/transfer_to(mob/new_character, var/force_key_move = 0)
+	if(martial_art)
+		var/mob/living/carbon/human/H = current //Martial arts can only be taught to humans, so it's safe to typecast here without worrying

```

Comment:
I'd prefer if you moved every thing inside this proc to a proc on the martial art
e.g.
```
if(martial_art)
    martial_art.mind_transfer(current, new_character)
```